integration/internal/network/ops.go

```diff
@@ -79,15 +79,20 @@ func WithOption(key, value string) func(*types.NetworkCreate) {
 }
 
 // WithIPAM adds an IPAM with the specified Subnet and Gateway to the network
-func WithIPAM(subnet, gateway string) func(*types.NetworkCreate) {
+func WithIPAM(driver, subnet, gateway, IPRange string) func(*types.NetworkCreate) {
```

Comment:
We may want to make this more optional (i.e. something like…)

```go
func WithIPAM(op ...func(*network.IPAMConfig)) func(*types.NetworkCreate) {
// …
}

// Using like
network.WithIPAM(Driver("foo"), Subnet(…), …)
```

But that could be done in a follow-up :wink:

Reply:
Thank you for the review. I will submit a PR for this.